src/plugins/i18nPolyfill.ts

```diff
@@ -1,18 +1,18 @@
 import companionTranslations from './companionTranslations';
 import gettextFactoryConst from './gettextFactory.const';
 
-const i18nModuleString = `
+const geti18nModuleString = (fallbackLocale: string) => `
   import languageTable from '\0lang:table';
   import gettextFactory from '\0lang:gettext-factory';
   import { locale } from 'user-settings';
-  const gettext = gettextFactory(languageTable, locale.language);
+  const gettext = gettextFactory(languageTable, locale.language, '${fallbackLocale}');
```

Comment:
I'm a bit paranoid. Let's escape the string just in case a `'` or other funny characters slip through.
```suggestion
  const gettext = gettextFactory(languageTable, locale.language, ${JSON.stringify(fallbackLocale)});
```